src/nupic/algorithms/Connections.cpp

```diff
@@ -37,6 +37,8 @@ using namespace std;
 using namespace nupic;
 using namespace nupic::algorithms::connections;
 
+static const Permanence EPSILON = 0.00001;
```

Comment:
Should this be some "global" constant? Which would define what we consider rounding error, as I'm defining the same in my example #890 and I'm sure it's used elsewhere.